src/pathwayvis/components/map/map.component.ts

```diff
@@ -186,10 +187,14 @@ class MapComponentCtrl {
     }, true);
 
     $scope.$on('$destroy', () => {
-      ws.close(this._mapOptions.getDataModelId());
+      this._connections.close(this._mapOptions.getDataModelId());
     });
   }
 
+  private _loaded() {
```

Comment:
I'm hoping to get all 

    this._mapOptions.setDataModel();
    this._mapOptions.setReactionData();
    this._mapOptions.setCurrentGrowthRate();

calls to mapOptions, and then there will be other ways to indicate that the first model is loaded.

Reply:
It is a better name haha